.travis.yml

```diff
@@ -13,14 +21,21 @@ jobs:
           - CasperLabs
       script:
         - make test
+        - make integration-tests
       after_success:
         - bash <(curl -s https://codecov.io/bash)
 
     - stage: unit test
       os: osx
       osx_image: xcode11.3
-      language: go
+      language: 
+        - go
+        - shell
```

Comment:
I am curious that why do we need the shell.

Reply:
Did as described at https://docs.travis-ci.com/user/languages/python/#running-python-tests-on-multiple-operating-systems
But let me try to get rid of that

Reply:
Oh, I got it, then how about leaving a related issue link as a comment?

Reply:
Works well. Resolved

Reply:
Changed as what you said against the doc, but it works well :)